com.unity.render-pipelines.universal/Runtime/2D/Renderer2DData.cs

```diff
@@ -93,6 +96,21 @@ public enum Renderer2DDefaultMaterialType
         internal Vector3 transparencySortAxis => m_TransparencySortAxis;
         internal uint lightRenderTextureMemoryBudget => m_MaxLightRenderTextureCount;
 
+        // Added for Tests.
+        internal bool enableBatching
+        {
+            get { return m_EnableBatching; }
+            set { m_EnableBatching = value; }
+        }
+
+        internal void SetLightBlendStyle(Light2DBlendStyle style, int i)
```

Comment:
what is this?

Reply:
Need to expose this for Playmode tests to set this.

Reply:
Do we still need this now that there's no more restriction on what blend style can work with batching?

Reply:
Fixed in c758a3b9ffc184b30ba26bef8fd0162b65f4725d